### DISCUSSION TITLE
Loss

### DISCUSSION THREAD

**Discussion by ghost** [STVIR/pysot#293] [General]
can you give the Loss in detail?
thanks

**Comment by ZhiyuanChen** [comment]
I'm having troubles understanding your question. 
What do you mean by Loss in detail?

**Comment by ghost** [comment]
siamrpn++的Loss公式是什么？




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Zhiyuan Chen"<notifications@github.com&gt;; 
发送时间: 2020年2月22日(星期六) 上午10:54
收件人: "STVIR/pysot"<pysot@noreply.github.com&gt;; 
抄送: "陈柏霖"<2549853330@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [STVIR/pysot] Loss (#293)




I'm having troubles understanding your question.
 What do you mean by Loss in detail?

**Comment by ZhiyuanChen** [comment]
loss = cls_loss * cls_weight + reg_loss * reg_weight + disc_loss * disc_weight

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing this issue for now. 
Please create a new issue should you have other questions.
